build.gradle.kts

```diff
@@ -23,14 +23,15 @@ sourceSets {
             // We need the entire source directory here, otherwise we get a
             // "package is empty or does not exist" error during compilation.
             srcDir("src/main/java")
+            include("org.jetbrains.annotations")
             compileClasspath = sourceSets.main.get().compileClasspath
         }
     }
 }
 
 dependencies {
     implementation("cafe.cryptography:curve25519-elisabeth:0.1.0")
-
+    implementation("org.jetbrains:annotations:16.0.2")
```

Comment:
This library requires [JDK 8 or later](https://www.jetbrains.com/help/idea/annotating-source-code.html), but `ed25519-elisabeth` supports JDK 7. I'll re-evaluate at some point whether to drop JDK 7, but for now this should use `annotations-java5`.

Reply:
Oh wait, I mis-read the page. The backwards-compatible version is only required if the library is being compiled with a JDK prior to 8, but `ed25519-elisabeth` already requires JDK 9 or later to compile. So it's fine as-is.